README.md

```diff
@@ -64,92 +64,36 @@ Example:
 $ ./execute_tests.sh -p
 ```
 
-Also, it is possible to execute a single Molecule test suite by supplying a Molecule path to the `-m` flag:
+Execute a single Molecule test suite by supplying a Molecule path to the `-m` flag:
 ```
 $ ./execute_tests.sh -p -m molecules/molecule-rpc-openstack-post-deploy
 ```
 
-Virtualenv Deployment
----------------------
-
-The creation of the python virtualenv is handled as part of the
-`execute_tests.sh` script. It is usually more convenient to execute this
-script with a bogus module path or to abort the script after the virtualenv is
-deployed.
-
-TODO: [ASC-1185](https://rpc-openstack.atlassian.net/browse/ASC-1185) Create an
-option in `execute_scripts.sh` to terminate without executing molecule tests.
-
-If you wish to deploy the virtualenv manually, here are the steps to do so.
-
-On the Deployment Host, create a python virtualenv and populate it using the included
-`requirements.txt` and `constraints.txt` files. This will install `molecule` and
-`ansible` into the virtualenv.
-
-Example
+Skip execution of the Molecule converge stage by supplying the `-c` flag:
```

Comment:
I think the flag should have a 'skip' action description? We have '-m' for executing 'molecule' so it is a bit confusing if '-c' for skipping 'converge'. 
Suggestion: '-sc, '-skip-c', '-skip-converge',